app/src/main/java/org/jellyfin/androidtv/util/UriExtensions.kt

```diff
@@ -0,0 +1,24 @@
+package org.jellyfin.androidtv.util
+
+import android.content.ContentResolver
+import android.net.Uri
+import androidx.annotation.AnyRes
+import org.jellyfin.androidtv.TvApp
+
+/**
+ * Get the uri for a given resource
+ */
+fun uriFromResourceId(@AnyRes resid: Int): Uri {
```

Comment:
Seems like this could be used to replace this [method](https://github.com/jellyfin/jellyfin-androidtv/blob/master/app/src/main/java/org/jellyfin/androidtv/util/ImageUtils.java#L280-L289) or perhaps be made to share some common code given that it returns a `String` and this returns a `Uri`.

Reply:
Good one, I changed the method in the ImageUtils class to properly specify the path (it was missing the "resource type name") and used that one. I tested it and it still works for the other usages as well.